.github/workflows/ui-tests.yml

```
@@ -0,0 +1,96 @@
+      # Add caching for faster builds
+      - name: Cache pnpm modules
+        uses: actions/cache@v3
+        with:
+          path: ~/.pnpm-store
+          key: ${{ runner.os }}-pnpm-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-
+          
+      - name: Install dependencies
+        run: pnpm install"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)